WeakAuras/GenericTrigger.lua

```diff
@@ -852,41 +851,41 @@ function HandleEvent(frame, event, arg1, arg2, ...)
   end
   if (event == "PLAYER_ENTERING_WORLD") then
     timer:ScheduleTimer(function()
-      WeakAuras.StartProfileSystem("generictrigger WA_DELAYED_PLAYER_ENTERING_WORLD");
+      Private.StartProfileSystem("generictrigger WA_DELAYED_PLAYER_ENTERING_WORLD");
       HandleEvent(frame, "WA_DELAYED_PLAYER_ENTERING_WORLD");
-      WeakAuras.CheckCooldownReady();
-      WeakAuras.StopProfileSystem("generictrigger WA_DELAYED_PLAYER_ENTERING_WORLD");
+      Private.CheckCooldownReady();
+      Private.StopProfileSystem("generictrigger WA_DELAYED_PLAYER_ENTERING_WORLD");
       if not WeakAuras.IsClassic() then
-        WeakAuras.PreShowModels() -- models are disabled for classic
+        Private.PreShowModels() -- models are disabled for classic
       end
```

Comment:
Not the purpose of this PR, but shouldn't this not even exist anymore since models were re-enabled for Classic?